apps/research-service/src/site-polygons/site-polygons.controller.ts

```diff
@@ -365,6 +366,34 @@ export class SitePolygonsController {
     return document.addIndex(indexData);
   }
 
+  @Patch("status/:status")
+  @ApiOperation({
+    operationId: "updateSitePolygonStatus",
+    summary: "Update the status of a site polygon",
+    description: "Update the status of a site polygon"
```

Comment:
These descriptions make it sound like this endpoint operates on a single polygon.